src/libraries/Microsoft.Extensions.Logging.Console/src/JsonConsoleLogFormatter.cs

```diff
@@ -0,0 +1,168 @@
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Extensions.Logging.Console
+{
+    internal class JsonConsoleLogFormatter : ConsoleFormatter, IDisposable
+    {
+        private IDisposable _optionsReloadToken;
+
+        public JsonConsoleLogFormatter(IOptionsMonitor<JsonConsoleFormatterOptions> options)
+            : base (ConsoleFormatterNames.Json)
+        {
+            FormatterOptions = options.CurrentValue;
+            ReloadLoggerOptions(options.CurrentValue);
+            _optionsReloadToken = options.OnChange(ReloadLoggerOptions);
+        }
+
+        public override void Write<TState>(in LogEntry<TState> logEntry, IExternalScopeProvider scopeProvider, TextWriter textWriter)
+        {
+            LogLevel logLevel = logEntry.LogLevel;
+            string category = logEntry.Category;
+            int eventId = logEntry.EventId.Id;
+            string message = logEntry.Formatter(logEntry.State, logEntry.Exception);
+            Exception exception = logEntry.Exception;
+            const int DefaultBufferSize = 1024;
+            var output = new ArrayBufferWriter<byte>(DefaultBufferSize);
+            using (var writer = new Utf8JsonWriter(output, FormatterOptions.JsonWriterOptions))
+            {
+                writer.WriteStartObject();
+
+
+                string timestamp = null;
+                var timestampFormat = FormatterOptions.TimestampFormat;
+                if (timestampFormat != null)
+                {
+                    var dateTime = FormatterOptions.UseUtcTimestamp ? DateTime.UtcNow : DateTime.Now;
+                    timestamp = dateTime.ToString(timestampFormat);
+                }
+                writer.WriteString("timestamp", timestamp);
+                writer.WriteNumber("eventId", eventId);
+                writer.WriteString("logLevel", GetLogLevelString(logLevel));
+                writer.WriteString("category", category);
+                writer.WriteString("message", message);
+
+                if (exception != null)
+                {
+                    writer.WriteStartObject("exception");
+                    writer.WriteString("message", exception.Message.ToString());
+                    writer.WriteString("type", exception.GetType().ToString());
+                    writer.WriteStartArray("stackTrace");
+                    if (exception?.StackTrace != null)
+                    {
+                        foreach (var stackTraceLines in exception?.StackTrace?.Split(Environment.NewLine))
+                        {
+                            JsonSerializer.Serialize<string>(writer, stackTraceLines);
```

Comment:
Don't use the serializer for this.  See an example here: https://github.com/dotnet/runtime/blob/0bbdc8e2fefd52bdae62a071ec1b4fdf952f425f/src/libraries/System.Text.Json/tests/Utf8JsonWriterTests.cs#L708-L713
/cc @layomia

Reply:
```suggestion
                            writer.WriteStringValue(stackTraceLines);
```